saplings/contracts/images/contracts_logo.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="42" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Where is this logo from? If it was created by cargill we should add the cargill header

Reply:
This is the circuits logo which has been renamed for use In the contracts sapling.